docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+With no real ability to reconcile these creative differences and with such different goals in mind, it was clear that the projects needed to go their own way. The group that wanted to see Atom reach its full potential split off and started work on what would, after a [community poll](https://discord.com/channels/737254137690980372/803109315480125514/990452760661098557), become **Pulsar**. Pulsar was chosen to represent the rebirth of the project, something that almost died but that still emits pulses of energy and can feed on energy pumped into it.
+
+A new Discord server was created for the new community as well as a GitHub organisation in order to organise for the work to come.
+
+### Salvage operations begin
+
+<img src="./assets/atom-packages.png" height="200" />
+
+One of the biggest problems was that time was not on our side. In under six months we would be losing the original Atom.io website and, most importantly, the package registry. The Atom.io package backend was not an open source project like the rest of the editor but was essential in keeping a list of published packages for Atom and serving them to its inbuilt package manager, APM. This is where [@confused-techie] was particularly instrumental in not only the creation of [a tool](https://github.com/confused-Techie/AtomPackagesArchive) to archive the entire package backend from Atom.io but also the, from scratch, re-implementation of the backend service needed to serve the reborn editor.
+
+This was no mean feat and is too much to go into in this blog post, but, thankfully, this is all detailed in our [first ever blog post](https://pulsar-edit.dev/blog/20221127-confused-Techie-SunsetMisadventureBackend.html) if you really want to understand the full scale of the undertaking that was the archival and re-implementation of the Atom backend to turn it into the Pulsar Package Registry.
+
+Of course, there was the main technical challenge, but this wasn't the only obstacle; we had other things that had to be taken into account, like what to do about packages with [no license](https://pulsar-edit.dev/blog/20230319-confused-Techie-HowLicenseNoneDeletedPackages.html), or custom licenses specifically for Atom.io? We also had to deal with a huge amount of spam from Vietnamese and Thai gambling sites that had been hammering the original backend in its final days with no moderation. To indicate the scale of the issue, 12k packages were archived compared to 405k just before atom.io finally shut down.
```

Comment:
```suggestion
There were other challenges, too — like what to do about packages with [no licence](https://pulsar-edit.dev/blog/20230319-confused-Techie-HowLicenseNoneDeletedPackages.html), or custom licenses designed specifically for Atom.io. We also had to deal with a huge amount of spam from Vietnamese and Thai gambling sites that had been hammering the original backend. Out of around 405,000 packages, roughly 12,000 packages were archived for reasons of license incompatibility, license absence, or spam.
```

Earlier you were using “licence,” now I see “license” — happy to standardize on either spelling here. :)

Reply:
Yeah it is meant to be whatever the more common American one is, I find it tricky to make myself type that way consistently as "Licence" comes more naturally... I'll accept the change and do a replacement later.